content/en/blog/_posts/2019-09-23-san-diego-contributor-summit

```diff
@@ -0,0 +1,75 @@
+---
+title: "Contributor Summit San Diego Registration Open!"
+date: 2019-09-23
```

Comment:
hello @parispittman .
Is the front matter for this page missing the layout field, such as:

```
---
layout: blog
title: "Kubernetes 1.16: Custom Resources, Overhauled Metrics, and Volume Extensions"
date: 2019-09-18
slug: kubernetes-1-16-release-announcement
---
```
Would it be helpful to create a template to populate the required front matter for blog pages?

Reply:
I think this is the only thing missing from the blog right now @parispittman, then we can roll it out.